Commands/PM/giveaway.js

```diff
@@ -38,18 +38,7 @@ module.exports = (bot, db, config, winston, userDocument, msg, suffix, commandDa
 											}).then(() => {
 												bot.awaitMessage(msg.channel.id, msg.author.id, message => {
 													if(config.yes_strings.indexOf(message.content.toLowerCase().trim()) > -1) {
-														const winner = Giveaways.end(bot, svr, serverDocument, ch, channelDocument);
-														msg.channel.createMessage({
-															embed: {
-                                                                author: {
-                                                                    name: bot.user.username,
-                                                                    icon_url: bot.user.avatarURL,
-                                                                    url: "https://github.com/GilbertGobbels/GAwesomeBot"
-                                                                },
-                                                                color: 0x00FF00,
-																description: `Alright, giveaway ended.\n${winner ? (`The winner was **@${bot.getName(svr, serverDocument, winner)}**`) : "I couldn't choose a winner for some reason tho 😕"}`
-															}
-														});
+														const endgiveaway = Giveaways.end(bot, db, svr, ch);
```

Comment:
Maybe you could capitalize Giveaway? *thats how I would do it and you don't have to* 😛